src/plugins/index_pattern_management/public/components/edit_index_pattern/indexed_fields_table/__snapshots__/indexed_fields_table.test.tsx.snap

```diff
@@ -1,5 +1,146 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`IndexedFieldsTable IndexedFieldsTable with rollup index pattern should render normally 1`] = `
```

Comment:
I am not sure this test tests what we really wanted to test here

Reply:
Its testing the params used for rendering which I think is good enough and useful. Do you have a better method?